lib/resources.rb

```diff
@@ -160,7 +160,7 @@ def beta_tag(action: false, known_issues: nil)
           beta += '</ul>'
         end
 
-        beta + '</div>'
+        "#{beta}</div>"
```

Comment:
This was not there in the old PR. Is this needed for capability statement?

Reply:
Actually this was autocorrected, I have removed this.